### PR TITLE
feat(ls-login): ls-login-button component

### LINKED ISSUE #66: feat: package for LS login button component
This feature recommends abstraction of login flow into [Web-Component](https://developer.mozilla.org/en-US/docs/Web/API/Web_components) by creating a button that handles all the logic for authentication and authorization.

### PR DESCRIPTION
**Pull request by tanya17-05**
fix #66 

## Description
initializes package for ecc-client-ga4gh-ls-login web component

- [x] My code follows the [contributing guidelines][contributing] of this
      project, including, in particular, with regard to any style guidelines
- [x] The title of my PR complies with the [Conventional Commits
      specification][conventional-commits]; in particular, it clearly indicates
      that a change is a breaking change
- [x] I am aware that all my commits will be squashed into a single commit,
      using the PR title as the commit message.
- [x] I have performed a self-review of my own code
- [x] I have commented my code in hard-to-understand areas
- [x] I have updated the user-facing documentation to describe any new or
      changed behavior
- [x] I have added type annotations for all function/class/method interfaces
      or updated existing ones
- [x] I have provided [JSDoc][jsdoc] block tags for all
      functions/classes/methods or updated existing ones
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature
      works
- [x] New and existing unit tests pass locally with my changes
- [x] I have not reduced the existing code coverage

[all-contributors-bot]: https://allcontributors.org/docs/en/bot/overview
[all-contributors-types]: https://allcontributors.org/docs/en/emoji-key
[contributing]: CONTRIBUTING.md
[conventional-commits]: https://www.conventionalcommits.org/en/v1.0.0/
[jsdoc]: https://jsdoc.app/index.html

### CODE REVIEW

**Comment by JaeAeich** [comment]
@tanya17-05 There are some issues in the FAST config for test, for now, either change the test script to null or use a different config (I did the later and @ltwmori fid the former) :+1:

**Review by uniqueg — COMMENTED** [review]
@tanya17-05: Thanks a lot. Given that I can only do a high-level review: Could you please add some description to the PR title and ideally include some link where I can try this out?

File: packages/ecc-client-ga4gh-ls-login/README.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
+# ecc-client-ga4gh-ls-login Components
+
+TRS component
```
Comment (git-anurag-hub) [inline packages/ecc-client-ga4gh-ls-login/README.md:3]: Correct this please! Also write some docs & scripts!

File: packages/ecc-client-ga4gh-ls-login/package.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    "description": "Lifescience login button component",
+    "private": true,
+    "type": "module",
+    "version": "0.1.0",
+    "author": "",
+    "license": "ISC",
+    "repository": "",
+    "scripts": {
+        "build": "webpack --config=./webpack.prod.cjs",
+        "eslint": "eslint . --ext .ts",
```
Comment (git-anurag-hub) [inline packages/ecc-client-ga4gh-ls-login/package.json:12]: Where in the `main` property? Please add it!

**Review by git-anurag-hub — CHANGES_REQUESTED** [review]
Hey @tanya17-05, I have pointed out some of the main & vital changes in the package! Please have a look & fix the configuration!

**Comment by git-anurag-hub** [comment]
Closing this due to inactivity!

**Pull request closed without merging**